ee/tabby-webserver/src/handler.rs

```diff
@@ -22,6 +25,7 @@ pub async fn attach_webserver(
     ui: Router,
     logger: Arc<dyn RawEventLogger>,
     code: Arc<dyn CodeSearch>,
+    conf: Arc<Config>,
 ) -> (Router, Router) {
     let ctx = create_service_locator(logger, code).await;
     let schema = Arc::new(create_schema());
```

Comment:
You might create Vec<Repositories> as `Arc` here, or create a wrapper struct, e.g
```rust
struct ResolveState {
  repositories: Vec<Repositories>
}
```

You can easily attach find_repository logic to this struct